src/gcs/gcsService.ts

```diff
@@ -54,20 +54,17 @@ export class GcsService {
    * Thin wrapper around storage.object.list
    * @see https://cloud.google.com/storage/docs/listing-objects#rest-list-objects
    */
-  static async list({ prefix, bucket, prefixCheck }: { prefix: string; bucket: string; prefixCheck: string }) {
+  static async list({ prefix, bucket }: { prefix: string; bucket: string }) {
```

Comment:
Could you explain what the prefixCheck attribute is for? I didn't quite understand the purpose of it @Shubha-accenture